src/main/java/DayFour.java

```diff
@@ -5,21 +5,25 @@
 public class DayFour {
     public String findInputs(Integer total, String listOfCodes) {
         List<String> codesList = Arrays.asList(listOfCodes.split(","));
+//indentation and typing
```

Comment:
It seems like a small thing but indentation is important, it will drive you and your fellow developers crazy if you don't get it right.  Also can we come up with a better name than `toCheck` and do we thing that `String` is the best type for it?